prefect_dbt/cloud/jobs.py

```diff
@@ -157,9 +157,9 @@ def trigger_job_run_flow():
     logger.info(
         f"Run successfully triggered for job with ID {job_id}. "
         "You can view the status of this run at "
-        f"https://{dbt_cloud_credentials.domain}/#/accounts/"
+        f"\033[1;34;40mhttps://{dbt_cloud_credentials.domain}/#/accounts/"
```

Comment:
What would it look like in UI / terminal if a space is added in between `\033[1;34;40m https...`